test/setup/iam.tf

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  int_required_proj_roles = [
+    "roles/owner",
```

Comment:
I think roles/owner overlaps with lots of roles below?

Reply:
I don't think so

Reply:
No it's not

Reply:
Actually, most of these roles are completely covered by the owner role, but the distinction of the roles in the README are also ambiguous so let's leave it for now.